src/www/data.njk.json

```diff
@@ -45,23 +45,35 @@
         {
           "name": "Buttons"
         },
+        {
+          "name": "Checkboxes"
+        },
         {
           "name": "Content card"
         },
         {
           "name": "Content tiles"
         },
+        {
+          "name": "Date Input"
+        },
         {
           "name": "Details"
         },
         {
           "name": "Disruption indicators"
         },
+        {
+          "name": "Dropdown"
+        },
         {
           "name": "File download"
         },
         {
-          "name": "Form elements"
+          "name": "File upload"
+        },
+        {
```

Comment:
Whilst looking into the header for the docs and adding a third tier. Do you think it'd be possible to do for form elements? - @houbly

Reply:
@daylesalmon I've added this to a new issue